### PR TITLE
Thu/refactor cypress scenario selector

### PR DESCRIPTION
**Pull request by csm-thu**

### CODE REVIEW

**Review by ardvi — APPROVED** [review]
LGTM

**Review by esasova — APPROVED** [review]
LGTM :+1:

File: cypress/commons/actions/generic/ScenarioSelector.js [hunk 1]
```diff
@@ -30,6 +84,14 @@ function selectScenario(scenarioName, scenarioId) {
       });
     });
 }
+
 export const ScenarioSelector = {
+  checkValidationStatusInScenarioSelector,
```
[the remark this answers is not in the document]
Comment (csm-thu) [reply cypress/commons/actions/generic/ScenarioSelector.js:89]: Thank you for catching this, I'll fix it :slightly_smiling_face:

File: cypress/integration/brewery/ScenarioListOrder.spec.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ describe('check scenario sorted list after reload', () => {
   it('checks first scenario in the list after reload', () => {
     ScenarioSelector.selectScenario(ALL_ROOT_SCENARIOS[1].name, ALL_ROOT_SCENARIOS[1].id);
     route.browse('W-stbbdbrwry/scenario');
-    Scenarios.getScenarioSelectorInput().should('have.value', ALL_ROOT_SCENARIOS[0].name);
+    ScenarioSelector.getScenarioSelectorInput().should('have.value', ALL_ROOT_SCENARIOS[0].name);
```
[the remark this answers is not in the document]
Comment (csm-thu) [reply cypress/integration/brewery/ScenarioListOrder.spec.js:33]: Done, thank you :)

**Pull request merged**